ytfzf

```diff
@@ -654,16 +653,15 @@ interface_thumbnails () {
 	selected_id_file=$2
 
 	# Download thumbnails, only if they're not already downloaded
-	[ -z "$(ls -1 "$thumb_dir" 2>/dev/null)" ] && {
+	find "$thumb_dir" -maxdepth 0 -type d -empty | grep -q . && {
```

Comment:
-maxdepth isn't [posix compliant](https://pubs.opengroup.org/onlinepubs/9699919799/)
I had a check that looked like this, but avoided it because of globbing and possible weird behavior.
```sh
set +f
ls "$thumb_dir"/* > /dev/null 2>&1 || { ...
```

Reply:
`-maxdepth` can be removed in this case and we will still have the same result, unless we expect to have have another empty directory inside `$thumb_dir`.
If we grep the name of the directory it will possibly solve even that issue:

```shell
find "$thumb_dir" -type d -empty | grep -q "$thumb_dir" && {
```
If this is okay, I'll push it

Reply:
Yes this is good.